src/components/MovieDetails/MovieDetails.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import { MovieDetails } from "./MovieDetails";
+import { MovieContext } from "@context/MovieContext";
+import { moviesList } from "@shared/constants";
+
+describe("MovieDetails", () => {
+  it("should be rendered correctly when a movie is selected", () => {
+    render(
+      <MovieContext.Provider value={{ selectedMovie: moviesList[0] } as any}>
```

Comment:
Please do not use `any`. You can create enum for example for every genre and use it or try to find another way.

Reply:
I saw any in another places as well. So I know it's not a TS course, but TS use with React 90% I believe, would be better avoid any. It's not like mandatory according to the task.

Reply:
Removed any